policyengine_us/variables/gov/states/ct/tax/income/subtractions/ct_social_security_benefit_adjustment.py

```diff
@@ -0,0 +1,36 @@
+from policyengine_us.model_api import *
+
+
+class ct_social_security_benefit_adjustment(Variable):
+    value_type = float
+    entity = TaxUnit
+    unit = USD
+    label = "Connecticut social security benefit adjustment"
+    reference = (
+        # Connecticut General Statutes, Chapter 229, Sec. 12-701, (20), (b), (x), (iii) and (iv)
+        "https://www.cga.ct.gov/current/pub/chap_229.htm#sec_12-701"
+        # 2022 Form CT-1040 Connecticut Resident Income Tax Return Instructions
+        "https://portal.ct.gov/-/media/DRS/Forms/2022/Income/2022-CT-1040-Instructions_1222.pdf#page=24"
+    )
+    definition_period = YEAR
+    defined_for = StateCode.CT
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.ct.tax.income.subtractions.social_security
+        filing_status = tax_unit("filing_status", period)
+
+        ss_rate = p.rate
+        # Line 41, Part A and Part B
+        total_ss = add(tax_unit, period, ["social_security"])
```

Comment:
CT only allows subtracting taxable SS (the portion included in federal AGI)
```suggestion
        us_taxable_ss = tax_unit("tax_unit_taxable_social_security", period)
```